### PR TITLE
Copy slow synthetic data streams to aux board

### PR DESCRIPTION
**Pull request by philippitts**
Closes https://github.com/brainflow-dev/brainflow/issues/555

### CODE REVIEW

File: src/board_controller/brainflow_boards.cpp [hunk 1]
```diff
@@ -79,8 +79,15 @@ BrainFlowBoards::BrainFlowBoards()
         {"package_num_channel", 0},
         {"timestamp_channel", 5},
         {"marker_channel", 6},
-        {"num_rows", 7},
-        {"other_channels", {1, 2, 3, 4}}
+        {"num_rows", 20},
+        {"other_channels", {1, 2, 3, 4}},
```
Comment (Andrey1994) [inline src/board_controller/brainflow_boards.cpp:83]: its better to add other channels to the end, they are meaningless most of the time and when you store data in a file they will be in the beginning but will not be interesting for the users.

Also, for all boards timestamp and marker channels are in the end, lets keep it consistent

**Review by Andrey1994 — CHANGES_REQUESTED** [review]
I think it will not work, in synthetic_board.cpp you need to fill it properly

**Comment by philippitts** [comment]
> I think it will not work, in synthetic_board.cpp you need to fill it properly

Resolved in https://github.com/brainflow-dev/brainflow/commit/0ab043c898871850060512a4b349aca82a9d8c4e

File: src/board_controller/brainflow_boards.cpp (hunk 1 above)
Comment (philippitts) [reply src/board_controller/brainflow_boards.cpp]: Resolved in https://github.com/brainflow-dev/brainflow/pull/556/commits/81ac4568e73e8a23eba69f37067a4fa5ee6f4649

**Comment by philippitts** [comment]
Assuming these fixes look okay, let me know if you'd like me to rebase and combine these commits into one before submitting for a merge.

**Comment by Andrey1994** [comment]
there is squash and merge option in github to create single commit, rebasing is not needed

**Review by Andrey1994 — APPROVED** [review]

**Pull request merged**